src/private/emscripten_api/emscripten_indoor_entity_api.js

```diff
@@ -3,7 +3,7 @@ function EmscriptenIndoorEntityApi(emscriptenApiPointer, cwrap, emscriptenModule
     var _emscriptenApiPointer = emscriptenApiPointer;
     var _emscriptenMemory = emscriptenMemory;
     var _indoorEntityApi_SetIndoorEntityPickedCallback = cwrap("indoorEntityApi_SetIndoorEntityPickedCallback", null, ["number", "number"]);
-    var _indoorEntityApi_SetHighlights = cwrap("indoorEntityApi_SetHighlights", null, ["number", "string", "number", "number", "number"]);
+    var _indoorEntityApi_SetHighlights = cwrap("indoorEntityApi_SetHighlights", null, ["number", "string", "number", "number", "number", "number"]);
```

Comment:
See comment in eegeo-mobile.  Did we not think it was safer to add an additional function with the additional arg rather than modifying this one?

Reply:
Yap. Sorry, have fixed with the latest commit.